librz/core/cmd_quit.c

```diff
@@ -32,6 +67,16 @@ static int cmd_Quit(void *data, const char *input) {
 	return -2;
 }
 
+RZ_IPI RzCmdStatus rz_cmd_force_quit_handler(RzCore *core, int argc, const char **argv) {
+	cmd_Quit(core, *argv);
```

Comment:
```suggestion
	cmd_Quit(core, argv[0] + 1);
```
The first character of argv[0] should be skip.